src/components/Service/getSections.js

```diff
@@ -7,7 +9,7 @@ export default workStage =>
       // arrays start with 0 so that's why the + 1
       const getCurrent = part => `section${capitalize(part)}${index + 1}`
       return {
-        id: workStage.id,
+        id: `${workStage.id}-${generate()}`,
```

Comment:
You don't need workStage at all here. Generate will suffice

Reply:
I wondered about that.